internal/repository/indb.go

```diff
@@ -130,3 +148,142 @@ func (l *InDBStorage) Ping() bool {
 
 	return true
 }
+
+func (l *InDBStorage) Urls(userid int) ([]LinkRecord, error) {
```

Comment:
```suggestion
func (l *InDBStorage) Urls(ctx context.Context, userid int) ([]LinkRecord, error) {
```
Контекст необходимо брать не из воздуха, а пробрасывать из хендлера, брать при этом из контекста запроса. Необходимо добавить этот параметр для всех методов сохранения, протащить в интерфейс и реализовать для всех стораджей. Там, где в методе контекст не используется, можно описать его, как `_ context.Context`

Reply:
поправил.